openslides_backend/services/auth/adapter.py

```diff
@@ -50,13 +47,6 @@ def is_equal(self, toHash: str, toCompare: str) -> bool:
     def is_anonymous(self, user_id: int) -> bool:
         return user_id == ANONYMOUS_USER
 
-    def create_authorization_token(self, user_id: int, email: str) -> str:
-        try:
-            response = self.auth_handler.create_authorization_token(user_id, email)
-        except AuthenticateException as e:
-            raise AuthenticationException(e.message)
-        return response.headers.get(AUTHORIZATION_HEADER, "")
-
```

Comment:
`openslides_backend/services/auth/interface.py` defines the interface for this class, including this method. You should remove the method from that place as well